com.vrcfury.vrcfury/Editor/VF/Feature/RemoveBlendshapeVerticesBuilder.cs

```diff
@@ -0,0 +1,171 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+using VF.Builder;
+using VF.Feature.Base;
+using VF.Inspector;
+using VF.Model.Feature;
+
+namespace VF.Feature {
+
+public class RemoveBlendshapeVerticesBuilder : FeatureBuilder<RemoveBlendshapeVertices> {
+    [FeatureBuilderAction]
+    public void Apply() {
+        var renderers = new List<SkinnedMeshRenderer>();
+        if (model.allRenderers) {
+            renderers.AddRange(avatarObject.GetComponentsInSelfAndChildren<SkinnedMeshRenderer>());
+        } else {
+            renderers.Add(model.renderer);
+        }
+
+        foreach (var renderer in renderers) {
+            var original = renderer.sharedMesh;
+            if (original == null)
+                continue;
+            var blendshapeIndex = original.GetBlendShapeIndex(model.blendshape);
+            if (blendshapeIndex < 0)
+                continue;
+
+            var mesh = MutableManager.MakeMutable(original, forceCopy: true);
```

Comment:
forceCopy does not need to be true here. It is safe to reuse a mutable copy that was already created earlier in the build.

Reply:
It needed to be true here because I call ClearMesh and for that I need a copy. But I'll refactor it to make that work without.